ddimport.js

```diff
@@ -429,12 +429,12 @@ class DDImporter extends Application
   static async DDImport(file, sceneName, fileName, path, fidelity, offset, extension, bucket, region, source) {
     if (path && path[path.length-1] != "/")
       path = path + "/"
-    if (path && path[0] != "/")
-      path = "/" + path
-    if (!path)
-      path = "/"
     let imagePath = path + fileName + "." + extension;
     if (source === "s3") {
+      if (path && path[0] != "/")
```

Comment:
I suspect you now actually want to manipulate imagePath directly to add the leading '/'.  Above that was being built from the modified path, but with this move you are modifying path but not using it to rebuild imagePath.  So changing to manipulating imagePath instead of path in this block is probably the way to go?

Reply:
You are absolutely right. Thanks